packages/styles/components/SplitButton.module.scss

```diff
@@ -0,0 +1,172 @@
+@import '~@venice/styles/core';
+
+.SplitButton {
+  display: inline-flex;
+  position: relative;
+}
+
+.caret {
+  margin-left: var(--space);
+  position: relative;
+
+  &::before,
+  &::after {
+    background-color: var(--white);
+    border-radius: 1px;
+    content: "";
+    display: block;
+    height: 10px;
+    position: absolute;
+    top: 25%;
+    transform: rotate(45deg);
+    width: 3px;
+  }
+
+  &::before {
+    left: 3px;
+  }
+
+  &::after {
+    right: 0;
+    transform: rotate(-45deg);
+  }
+}
+
+.dropdown {
+  @include reset();
+
+  align-items: center;
+  border-radius: var(--border-radius);
+  box-shadow: -6px 8px 15px rgba(0, 0, 0, 0.1);
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  position: absolute;
+  width: 100%;
```

Comment:
The problem's here is in a context of text of `button` is small that `dropdown` size. Call me that I explain it 😁